ravenpy/__init__.py

```diff
@@ -1,3 +1,7 @@
 """Top-level package for RavenPy."""
 
 from .__version__ import __author__, __email__, __version__  # noqa: F401
+
+
+class RavenPyDependencyError(BaseException):
+    pass
```

Comment:
What is the added value of this over ImportError ?

Reply:
If RavenPy is going to be a building block for other components (like Raven-WPS), it helps to know that the errors are coming from RavenPy. This just makes it more visible. Can replace with ImportError if we want.

Reply:
I don't have a strong opinion. Is this a common pattern ? 
I found this: https://stackoverflow.com/questions/27361427/how-to-properly-deal-with-optional-features-in-python
which uses ImportError as well with a custom message.

Reply:
It would be really easy to remove. I just don't want to see users installing the GIS dependencies one at a time (annoying). A custom error grabs attention, but so does `ImportError`

Reply:
Agreed. We want one clear message that says, please install RavenPy with such and such instructions for this to work. 
So let's keep it as is for now in order to move on.

Reply:
I've sadly removed the `RavenPyDependencyError` but the message is still present. Good enough,